tailwind.config.js

```diff
@@ -3,6 +3,7 @@ const colors = require('tailwindcss/colors');
 
 /* eslint-disable global-require */
 module.exports = {
+  mode: process.env.NODE_ENV === 'development' ? 'jit' : '',
```

Comment:
tailwind's new `Just In Time compiler` makes it so we don't have to worry about styles getting purged when developing locally.